src/main/resources/view/PersonListCard.fxml

```diff
@@ -28,9 +28,6 @@
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
       <FlowPane fx:id="tags" />
```

Comment:
Perhaps we can remove this line too? Same line of reasoning as my earlier comment.

Reply:
fixed, thanks